pkg/steps/testing.go

```diff
@@ -37,13 +76,97 @@ func (dl *DryLogger) AddObject(o runtime.Object) {
 	dl.objects = append(dl.objects, o)
 }
 
-// GetObjects returns the list of objects.
+func (dl *DryLogger) AddBuild(build *buildapi.Build) {
+	imageLabels := build.Spec.Output.ImageLabels
+	if len(imageLabels) > 0 {
+		sort.Slice(imageLabels, func(i, j int) bool {
+			return imageLabels[i].Name < imageLabels[j].Name
+		})
+	}
+
+	if images := build.Spec.Source.Images; len(images) > 0 {
+		for i, image := range images {
+			build.Spec.Source.Images[i].From.Name = trimSHA256(image.From.Name)
+		}
+
+	}
+
+	dl.AddObject(build.DeepCopyObject())
+}
+
+func (dl *DryLogger) AddImageStreamTag(ist *imageapi.ImageStreamTag) {
+	if tag := ist.Tag; tag != nil {
+		if tag.From != nil {
+			ist.Tag.From.Name = trimSHA256(ist.Tag.From.Name)
+		}
+	}
+	dl.AddObject(ist.DeepCopyObject())
+}
+
+func trimSHA256(value string) string {
+	reg := regexp.MustCompile("@sha256:[0-9a-f]{64}")
+	return reg.ReplaceAllString(value, "")
```

Comment:
nit: to make it clearer to someone reading, we could replace with `@sha256:SHA` or something

Reply:
addressed